spec/rubocop/cli/cli_options_spec.rb

```diff
@@ -1006,67 +1006,72 @@ def f
     end
 
     after do
+    end
```

Comment:
Remove this `after` 😉

Reply:
@backus I think you need to create a cop for that. ;-) Or we should do it. :D

Reply:
Probably it should be us.